cmd/vicadmin/vicadm.go

```diff
@@ -183,6 +183,13 @@ func newBytesEntry(name string, b []byte) entry {
 	}
 }
 
+type versionReader string
+
+func (path versionReader) open() (entry,error) {
+	defer trace.End(trace.Begin(string(path)))
```

Comment:
Minor, but I think the `trace` is unnecessary here, since this is not doing anything particularly complex (in contrast to `commandReader`, for example).